blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/OrganizationFactoryImpl.java

```diff
@@ -32,7 +38,17 @@ public OrganizationFactoryImpl() {
     }
 
     public OrganizationFactoryImpl(String name) {
-        this.instance = new OrganizationImpl(name, Jenkins.get());
+        if (ROOT_FOLDER_NAME != null) {
+            ItemGroup<?> root = Jenkins.get().getItemByFullName(ROOT_FOLDER_NAME, ItemGroup.class);
+            if (root != null) {
+                this.instance = new OrganizationImpl(name, root);
```

Comment:
Does not compile I think?
```suggestion
            var root = Jenkins.get().getItemByFullName(ROOT_FOLDER_NAME);
            if (root instanceof ItemGroup<?> group) {
                this.instance = new OrganizationImpl(name, group);
```

Reply:
> Does not compile I think?

possibly - BO doesn't compile on windows as all the javascript stuff fails.  VS code said "this is fine" but well that's not an IDE..